src/hooks/useArray.ts

```diff
@@ -82,14 +90,27 @@ export const useArray = <T>(
     },
     [items]
   );
-
+  console.log(isDirty);
```

Comment:
This shouldn't be making regular PR appearances

```suggestion
```

Reply:
941192c